### PR TITLE
[ENH] Robust tensor fitting

### PR DESCRIPTION
**Pull request by 36000**

### CODE REVIEW

File: AFQ/models/dti.py [hunk 1]
```diff
@@ -6,19 +6,86 @@
...
+            gtab,
+            fit_method="RT",
+            sigma=sigma)
     return dtimodel.fit(data, mask=mask)
 
 
 def fit_dti(data_files, bval_files, bvec_files, mask=None,
-            out_dir=None, file_prefix=None, b0_threshold=50):
+            out_dir=None, file_prefix=None, b0_threshold=50,
+            rtf=False):
```
Comment (arokem) [inline AFQ/models/dti.py:88]: Looks like this key-word argument is not being used

Comment (arokem) [reply AFQ/models/dti.py:88]: Also, can we use something more explicit? For example `restore=False` instead of `rtf`?

**Review by arokem — COMMENTED** [review]
Overall looks good. Have you had a chance to run this with some prek data?

File: AFQ/models/dti.py (hunk 1 above)
Comment (36000) [reply AFQ/models/dti.py:88]: To include restore in this function, we need to write another function that allows us to load multiple bval and bvec files then merge them together, I think. Which I have not done yet - the AFQ object does not use this function, but I think the examples do.

**Comment by 36000** [comment]
I ran this on 8 subjects and the results are inconclusive. I am now running it on all subjects (prek doesn't take as long as HCP) to see if we see the same problems.

File: AFQ/models/dti.py (hunk 1 above)
Comment (arokem) [reply AFQ/models/dti.py:88]: I think `ut.prepare_data` does what you want? It's the first thing this function calls. 

In general, as you noted, this function is not directly called by the API. 

When we began the project, @jyeatman and folks in his team wanted to be able to fit DIPY models from the command line, which is why we still have a bunch of CLIs that do that. I wonder whether we should remove functionality directly related just to that. I believe that DIPY now has CLIs that do all these things. @jyeatman : WDYT?

**Comment by 36000** [comment]
@arokem for now maybe we can ignore fit_dti. Other than that, this should be ready to merge. I just rebased it on master.

**Comment by arokem** [comment]
Looks good!

**Pull request merged**